specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1797 @@
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "The operation status identifier. It should match what is used to GET the operation resource.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "name": {
+          "description": "It must match the last segment of the id field, and will typically be a GUID / system generated value.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "type": {
+          "description": "The type.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "$ref": "#/definitions/RequestState",
+              "readOnly": true
+            },
+            "limit": {
+              "type": "integer",
+              "format": "int64",
+              "description": "The new quota limit for the subscription. The incremental quota will be assigned from pre-approved groupQuota."
+            }
+          }
+        }
+      }
+    },
+    "SubscriptionQuotaAllocationRequestList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "Assigned Group Quota to subscriptions.",
+          "items": {
+            "$ref": "#/definitions/SubscriptionQuotaAllocationRequest"
+          }
+        },
+        "nextLink": {
+          "description": "The URL to use for getting the next set of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SubscriptionGroupQuotaAssignment": {
+      "type": "object",
+      "description": "MGId the source of groupQuota.",
+      "additionalProperties": {
+        "type": "integer",
+        "format": "int64",
+        "description": "Group Quota assigned to subscription."
+      }
+    },
+    "SubscriptionGroupQuotaAssignmentList": {
+      "type": "array",
+      "description": "Assigned Group Quota to subscriptions.",
+      "items": {
+        "$ref": "#/definitions/SubscriptionGroupQuotaAssignment"
+      },
+      "x-ms-identifiers": []
+    },
+    "SubscriptionQuotaLimit": {
+      "type": "object",
+      "description": "Subscription Quota limit.",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The resource name.",
+          "type": "string",
+          "readOnly": true
+        },
```

Comment:
Use TrackedResource/ProxyResource model definitions from common types, instead of defining these properties here. See: [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5).